reladomographql/src/main/java/com/gs/reladomo/graphql/ReladomoMutationFetcher.java

```diff
@@ -0,0 +1,107 @@
+/*
+ Copyright 2019 Goldman Sachs.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ */
+
+package com.gs.reladomo.graphql;
+
+import com.gs.fw.common.mithra.MithraTransactionalObject;
+import com.gs.fw.common.mithra.attribute.*;
+import com.gs.fw.common.mithra.finder.RelatedFinder;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+
+import java.util.Map;
+import java.sql.Timestamp;
+
+public class ReladomoMutationFetcher implements DataFetcher
+{
+    private final RelatedFinder finder;
+    private final String name;
+    private final String reladomoClassName;
+
+
+    public ReladomoMutationFetcher (String name, String reladomoClassName, RelatedFinder finder)
+    {
+        this.finder = finder;
+        this.name = name;
+        this.reladomoClassName = reladomoClassName;
+    }
+
+    @Override
+    public MithraTransactionalObject get (DataFetchingEnvironment environment)
+    {
+        //
+        // The graphql specification dictates that input object arguments MUST
+        // be maps.  You can convert them to POJOs inside the data fetcher if that
+        // suits your code better
+        //
+        // See http://facebook.github.io/graphql/October2016/#sec-Input-Objects
+        //
+        Map<String, Object> objectInputMap = environment.getArgument (this.name);
+        try
+        {
+            MithraTransactionalObject obj = (MithraTransactionalObject) Class.forName (reladomoClassName).newInstance ();

```

Comment:
doesn't work with temporal objects

Reply:
added 
businessDate: Timestamp! 
to the schema. added test